README.md

```diff
@@ -71,6 +71,30 @@ $ VERSION=<VERSION> \
     ./bin/package_extension.sh
 ```
 
+The name of the packaged extension will be `screenly-<PLATFORM>-extension-<VERSION>.zip`. For example, `screenly-chrome-extension-0.0.1.zip` or `screenly-firefox-extension-0.0.1.zip`.
+
+### Testing Packaged Extensions
+
+> [!NOTE]
+> You can also download the packaged extensions from the [GitHub Releases](https://github.com/Screenly/Browser-Extension/releases) page.
+
+#### Chrome
+
+- Extract the zip file to a folder.
+- Open Chrome and navigate to `chrome://extensions/`.
+- Click on **Load unpacked** and select the folder you extracted the zip file to.
+- The extension should now be loaded and ready to use.
+
+#### Firefox
+
+> [!IMPORTANT]
+> Add-ons installed from a `.zip` file will be uninstalled when Firefox is closed.
+> This means that you will need to re-install the extension every time you open Firefox.
+
+- Open Firefox and navigate to `about:debugging`.
```

Comment:
We should choose "`about:debugging#/runtime/this-firefox`" to access the "**Load Temporary Add-on**" option direclty; otherwise, the user needs to go to "`about:debugging`" and select the "**This Firefox"** button. 

![firfix](https://github.com/user-attachments/assets/c1df1967-9e8a-4f17-a90d-1e5ad112ca4d)